components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/CodeTokenResponseValidator.java

```diff
@@ -58,6 +64,10 @@ private static boolean isContainOIDCScope(String scope) {
     @Override
     public void validateRequiredParameters(HttpServletRequest request) throws OAuthProblemException {
 
+        if (!StringUtils.isBlank(request.getParameter(REQUEST_URI))) {
+            requiredParams = new ArrayList<>(Arrays.asList(CLIENT_ID, RESPONSE_TYPE, SCOPE, REQUEST_URI));
+            notAllowedParams.add(REQUEST);
```

Comment:
What where the initial values for these?

Reply:
Its `response_type`, `client_id` and `redirect_uri`. For PAR we need these four as mandatory and `request` shouldnt be there.

Reply:
scope is not a mandatory param in oauth/oidc why is it added here?

Reply:
I added as it is required in OB spec, but I understand it is not OIDC compliant. I will remove it.